src/latexify/codegen/function_codegen.py

```diff
@@ -363,6 +366,35 @@ def visit_Call(self, node: ast.Call) -> str:
         arg_strs = [self.visit(arg) for arg in node.args]
         return lstr + ", ".join(arg_strs) + rstr
 
+    def _generate_matrix(self, node: ast.Call) -> str:
```

Comment:
`_generate_ndarray` makes sense more I think.

```suggestion
    def _generate_ndarray(self, node: ast.Call) -> str:
```